app/views/about/_small_and_fast.html.erb

```diff
@@ -89,22 +89,22 @@
           <th class="right">Git</th>
           <th class="right">SVN</th>
         </tr>
-        <tr><td nowrap="">Commit Files (A)</td><td class="desc">Add, commit and push 113 modified files (2164+, 2259-)</td><td class="number"> 0.64</td><td class="number"> 2.60</td><td class="number">4x</td></tr>
-        <tr><td nowrap="">Commit Images (B)</td><td class="desc">Add, commit and push 1000 1k images</td><td class="number"> 1.53</td><td class="number">24.70</td><td class="number">16x</td></tr>
-        <tr><td nowrap="">Diff Current</td><td class="desc">Diff 187 changed files (1664+, 4859-) against last commit</td><td class="number"> 0.25</td><td class="number"> 1.09</td><td class="number">4x</td></tr>
-        <tr><td nowrap="">Diff Recent</td><td class="desc">Diff against 4 commits back (269 changed/3609+,6898-)</td><td class="number"> 0.25</td><td class="number"> 3.99</td><td class="number">16x</td></tr>
-        <tr><td nowrap="">Diff Tags</td><td class="desc">Diff two tags against each other (v1.9.1.0/v1.9.3.0 )</td><td class="number"> 1.17</td><td class="number">83.57</td><td class="number">71x</td></tr>
-        <tr><td nowrap="">Log (50)</td><td class="desc">Log of the last 50 commits (19k of output)</td><td class="number"> 0.01</td><td class="number"> 0.38</td><td class="number">31x</td></tr>
-        <tr><td nowrap="">Log (All)</td><td class="desc">Log of all commits (26,056 commits - 9.4M of output)</td><td class="number"> 0.52</td><td class="number">169.20</td><td class="number">325x</td></tr>
-        <tr><td nowrap="">Log (File)</td><td class="desc">Log of the history of a single file (array.c - 483 revs)</td><td class="number"> 0.60</td><td class="number">82.84</td><td class="number">138x</td></tr>
-        <tr><td nowrap="">Update</td><td class="desc">Pull of Commit A scenario (113 files changed, 2164+, 2259-)</td><td class="number"> 0.90</td><td class="number"> 2.82</td><td class="number">3x</td></tr>
-        <tr><td nowrap="">Blame</td><td class="desc">Line annotation of a single file (array.c)</td><td class="number"> 1.91</td><td class="number"> 3.04</td><td class="number">1x</td></tr>
+        <tr><td nowrap="">Commit Files (A)</td><td class="desc">Add, commit and push 113 modified files (2164+, 2259-)</td><td class="number">0.64</td><td class="number">2.60</td><td class="number">4x</td></tr>
+        <tr><td nowrap="">Commit Images (B)</td><td class="desc">Add, commit and push 1000 1&nbsp;kB images</td><td class="number">1.53</td><td class="number">24.70</td><td class="number">16x</td></tr>
```

Comment:
The use of `&nbsp` here over a regular space certainly makes sense. I wonder, though if "1000 1 kB" ends up being hard to read because of the proximity of "1000" and "1" separated by a space. Would "1-kB images" be easier? I guess it's typographically inconsistent with the others then. Most style guides would probably suggest "one" over "1", but that looks a bit awkward too. I dunno.

Reply:
It's incorrect to write "1-kB" or "one kB", since the unit symbols are only meant to be used with numbers and since they're symbols, the hyphen doesn't make sense to make it into an adjective. Most style guides would suggest "one thousand 1 kB" or "a thousand 1 kB". I think the latter would be the best solution here. Thoughts?

Reply:
Sure, the last one looks fine to me.